BierBuddy.Core/FindBuddies.cs

```diff
@@ -106,5 +106,36 @@ public int GetInterestsPoints(Visitor ClientVisitor, Visitor PotentialMatchVisit
             return points;
         }
 
+        public int GetAgeDelta(Visitor ClientVisitor, Visitor PotentialMatchVisitor)
+        {
+            int ageDelta;
+            //zorg er voor dar ageDelta niet negatief of 0 kan zijn
+            if (ClientVisitor.Age == PotentialMatchVisitor.Age)
+            {
+                ageDelta = 1;
+            }
+            else if (ClientVisitor.Age > PotentialMatchVisitor.Age)
+            {
+                ageDelta = ClientVisitor.Age - PotentialMatchVisitor.Age;
+            }
+            else
```

Comment:
deze 2 kan je combineren met een Math.abs(), dit returnt namelijk altijd een positief getal waardoor het niet meer uitmaakt of de grotere leeftijd eerder in de som komt.